noisecapture_protocol.html

```diff
@@ -37,219 +37,307 @@
 
                 <div class="col-lg-6">
                 <br>
-                   <p align="center"><img class="img-responsive" src="./assets/img/noise_party_protocol/qrcode.png" alt="NoiseCapture QrCode" height="200px" width="200px"></p>
+                   <p align="center"><img class="img-responsive" src="./assets/img/noisecapture/NoiseCapture_home_page.png" alt="NoiseCapture measures" width="300px"></p>
                 </div><!-- col-lg-6 -->
                 <br>
             </div><!-- row -->
             </div><!-- container -->
 
-             <!-- Screenshots -->
-             <div class="container" id="screenshots" name="screenshots">
+            <!-- Start measuring -->
+             <div class="container" id="start_measuring" name="start_measuring">
              <div class="row">
+                <h3 class="centered">Start measuring</h3>
+                <div class="col-lg-6">
+                <br>
+                   <p align="center"><img class="img-responsive" src="./assets/img/noisecapture/NoiseCapture_during_measure.png" alt="Start measuring" width="300px"></p>
+                </div><!-- col-lg-6 -->
+
+                <div class="col-lg-6">
+                <br>
+                    <p align="justify">Once you started the recording (you clicked on the dedicated icon), you obtain this interface (on the left), with the following informations:
+                    <ol>
+                        <li><b>dB value</b>: real time noise level (exprimed in dB(A)),</li>
+                        <li>Noise <b>statistics</b>: min, moy and max noise dB values,</li>
+                        <li>Measure's <b>duration</b>,</li>
+                        <li><b>GPS precision</b>, exprimed in meter,</li>
+                        <li>Noise representation <i>(see "Measuring modes" below)</i>,</li>
+                        <li><b>Stop recording</b> button: press it to stop the measure,</li>
+                        <li><b>Pause recording</b> button: press it to pause the measure.</li></li>
+                    </ol>
+                    </p>
+                </div><!-- col-lg-6 -->
+            </div><!-- row -->
+            </div><!-- container -->
+
+            <!-- Measuring modes -->
+            <div class="container" id="measures_mode" name="measures_mode">
+            <div class="row">
+                <h3 class="centered">Measuring modes</h3>
                 <div class="col-lg-6">
-                   <br>
-                   <p align="center"><img class="img-responsive" src="./assets/img/noise_party_protocol/Mesure_description.png" alt="Mesure description" height="450px" width="437px"></p>
+                <br>
+                    <p align="justify">During the recording, you have the possibility to change the representation mode. Here, by clicking on the dedicated tabs, you have the choice between:
+                    <ol>
+                        <li><b>Spectre</b>: the noise is represented thanks to frequencies bands,</li>
```

Comment:
Spectrum, Spectrogram

Reply:
the noise is displayed in a sonar mode.

Graph with two geometric dimensions: the horizontal axis represents time , the vertical axis is frequency; a third dimension (color) indicating the amplitude in dB(A)